test/test-TabTracker.js

```diff
@@ -0,0 +1,113 @@
+"use strict";
+
+const {before, after} = require("sdk/test/utils");
+const tabs = require("sdk/tabs");
+const ss = require("sdk/simple-storage");
+const {setTimeout} = require("sdk/timers");
+const {TabTracker} = require("lib/TabTracker");
+
+const NEWTAB_URL = "resource://activity-streams/data/content/activity-streams.html#/";
```

Comment:
Nit: this is duplicated in this file and the previous file. Not sure if it is worth exporting from one file and sharing, or trying to shim into webpack config or something.